modmail/__init__.py

```diff
@@ -26,7 +26,7 @@
 # file handler
 file_handler = logging.handlers.RotatingFileHandler(
     log_file,
-    maxBytes=5 * (2 ** 14),
+    maxBytes=5 * (2 ** 14),  # 81920 bytes, approximately 200 lines
```

Comment:
Alright. Now that we know what this is this is WAYYY too short. When I break a feature I generate more than 200 lines of output by accident, so I suggest we up this to 200k lines at least. That also lines up nicely with the megabyte barrier, which is where things go from easy upload to not easy upload, and certain tools start getting cumbersome. Just up this by `1000x` or 1024x if that's easier

Reply:
Done, upped it to 8MB